README.rst

```diff
@@ -360,6 +360,27 @@ Methods:
     a celery's task for example or to handle errors.
 
 
+Settings
+=============
+
+You can configure Django-Templated-Email by setting the following settings
+
+.. code-block:: python
+
+    TEMPLATED_EMAIL_FROM_EMAIL      # String containing the email to send the email from - fallback to DEFAULT_FROM_EMAIL  
+    TEMPLATED_EMAIL_BACKEND         # The backend that will send the email  
+    TEMPLATED_EMAIL_TEMPLATE_DIR    # The directory containing the templates, use '' if using the top level
+    TEMPLATED_EMAIL_FILE_EXTENSION  # The file extension of the template files
+    TEMPLATED_EMAIL_AUTO_PLAIN      # Disable the behavior of calculating the plain part from the html part of the email when `html2text <https://pypi.python.org/pypi/html2text>` is installed
+    TEMPLATED_EMAIL_PLAIN_FUNCTION  # Specify a custom function that converts from HTML to the plain part
+
+    # vanilla_django
+    TEMPLATED_EMAIL_DJANGO_SUBJECTS # Deprecated, prefer to use **{% block subject %}**
```

Comment:
You can hide this one from this part of the docs. Remove this line.

Reply:
Removed